exporter/exporter.go

```diff
@@ -267,6 +268,21 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
 			"server_threads":        "server_threads_total",
 			"long_lock_waits":       "long_lock_waits_total",
 			"current_client_thread": "current_client_thread",
+
+			// Redis Modules metrics
+			// RediSearch module
+			"search_number_of_indexes":   "search_number_of_indexes",
+			"search_used_memory_indexes": "search_used_memory_indexes_bytes",
```

Comment:
Thanks for changing this one, what about the others, do they all adhere to prom best practices re: the units? 

For instance: `search_total_ms_run` - if this is in milliseconds then the metric name should be in seconds and end in `_seconds`

Reply:
That's good question.
I don't know much details about metrics Search module provides and how to handle them.

On example of search_total_ms_run, how I can modify it in exporter from ms to sec?

Reply:
Ah, good point, that would require extracting the metric manually and then divide by 1000 to get to seconds.
Let's keep it simple and not do that.

Reply:
Actually, just re-checked adding metrics and I think it might make sense to add units to these 3 more, so I commit changes. I cannot convert ms to sec, but I think it's worth to move `ms` to the end.

```
"search_total_indexing_time": "search_total_indexing_time_ms",
"search_bytes_collected":     "search_collected_bytes",
"search_total_ms_run":        "search_total_run_ms",
```

Reply:
No, I think this is fine.